packages/nouns-webapp/src/components/NavWallet/index.tsx

```diff
@@ -69,7 +69,7 @@ const NavWallet: React.FC<NavWalletProps> = props => {
   const statePrimaryButtonClass = usePickByState(
```

Comment:
Good idea to move all these `usePickByState` into its own file. Great job on `colorResponsiveUIUtils.ts` 🔥